### PR TITLE
[DOCS] Removing datasource centric test_yaml_config doc

### PR DESCRIPTION
**Pull request by donaldheppner**
This doc was focused on using test_yaml_config with YAML datasource configurations. This is no longer required.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: docs/docusaurus/docs/terms/data_context.md
Comment (kwcanuck) [inline docs/docusaurus/docs/terms/data_context.md]:
```suggestion
A Data Context includes basic create, read, update, and delete (CRUD) operations for the core components of a Great Expectations deployment. This includes Datasources, Expectation Suites, and Checkpoints. In addition, a Data Context allows you to access and integrate Data Docs, Stores, Plugins, and so on.
```

image

image

image

**Review by kwcanuck — APPROVED** [review]

File: docs/docusaurus/docs/components/setup/link_lists/_setup_configurations.md
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/components/setup/link_lists/_setup_configurations.md:75]: Removing just this bit leaves us with an empty `<details>` tag.  We should remove the entire thing, not just the inside content.

File: docs/docusaurus/docs/terms/data_context.md
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/terms/data_context.md:116]: The entire section `### Interactively testing configurations from your Data Context` is about `test_yaml_config()` -- do we want to remove this full section, instead of just the part with the link to the `test_yaml_config()` guide?

**Review by Rachel-Reverie — APPROVED** [review]
I flagged two items that should get a second look, but I'm approving so that you won't be blocked on my re-review.  Everything other than those looked fine.

File: docs/docusaurus/docs/terms/data_context.md
Comment (donaldheppner) [reply docs/docusaurus/docs/terms/data_context.md:116]: good idea

**Pull request merged**
